src/docs/010_manual/20_install.adoc

```diff
@@ -153,6 +155,31 @@ In this case you can pass the image name via parameter `image` directly after `d
 .\dtcw.bat docker image <image_name> generateHTML
 ----
 
+[[add-environment-variables-to-docker]]
+=== Pass environment variables to docker containers
+To pass any environment variable to the docker container you can make use of an https://docs.docker.com/engine/reference/commandline/run/#env[environment file].
+The environment file must have the name `dtcw_docker_env.list` and must be located in the same folder as `dtcw`.
```

Comment:
Good work, thanks for your efforts, really appreciate that :) 
One thing came to my mind after having a first look, i would rather prefer `dtcw_docker.env` as a filename than `dtcw_docker_env.list`
Mainly because it is an `.env` file and if you look at how other tools handle such files, it is mostly `something.env` 
What do you think to adopt your PR to follow this covention?

Reply:
Hi @PacoVK,

I agree with you now 😄. I took the extension from an example - but have now seen that more use *.env.

If you still want to have this change - I can create another MR.

As long as it is done before the next release it will not break anyone's setup.

Reply:
Hi @PacoVK,

yes I was signed in with another account. I misread the comment of @rdmueller as already merged.

I upload a new change.

Reply:
File has been renamed to the proposed: `dtcw_docker.env`.

Reply:
Good work, thanks! :)